src/server/game/Entities/Player/Player.cpp

```diff
@@ -1017,6 +1032,9 @@ void Player::SetDrunkValue(uint8 newDrunkValue, uint32 itemId /*= 0*/)
 
 void Player::Update(uint32 p_time)
 {
+    if (!GetSession())
```

Comment:
does this skip some more stuff than what the !IsInWorld() check below skips ?

Reply:
This is an impossible condition.